src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -7966,26 +7966,6 @@ private struct REPARSE_GUID_DATA_BUFFER
             IntPtr OutBuffer, int nOutBufferSize,
             out int pBytesReturned, IntPtr lpOverlapped);
 
-#if !CORECLR
-
-        [DllImport("kernel32.dll", SetLastError = true, CharSet = CharSet.Unicode)]
-        private static extern IntPtr FindFirstFileName(
```

Comment:
Then `private static List<string> InternalGetTarget(string filePath)`  needs to be revisited as it basically only handles Unix platforms. Maybe just remove it, and change `public static IEnumerable<string> GetTarget(PSObject instance)` to directly handle `Windows/Unix` with `if UNIX/else`.